config/config.inc.php

```diff
@@ -206,15 +206,16 @@
 $config['remotebuzzer_enabled'] = false;
 $config['remotebuzzer_port'] = 14711;
 $config['remotebuzzer_pin'] = 40;
-$config['remotebuzzer_nodebin'] = '/usr/bin/node';
 $config['remotebuzzer_collagetime'] = '2'; // control time to distinguish picture from collage
 
 // PRESERVE EXIF
 $config['exiftool']['cmd'] = null;
 $config['exiftool']['msg'] = null;
 
+// NODE BINARY
+$config['nodebin']['cmd'] = null;
+
 // BACKUP/SYNC SCRIPT
-// Currently supported formats are : sdX, /dev/sdX and simply the label of the drive (not the manufacturer)
-// In the default config the script will only search for drives with the label photobooth!
-$config['sync_script_enabled'] = false;
-$config['sync_script_targets'] = ['photobooth']; //Default targets for the sync sctipt
+$config['synctodrive_enabled'] = false;
+$config['synctodrive_targets'] = 'photobooth'; //Default targets for the sync sctipt
```

Comment:
Why not use an array here? Adds complexity because we need to split with `;` later on

Reply:
Rational is that if this array of parameters is to be configured in the admin GUI of photobooth, there is no easy way to integrate an array for users to flexibly enter any number of values there. I can see three options:

1. In the admin GUI add three separate variables as string values, where each is a pattern to match. This limits to three patterns and the script then needs to build an array from three distinct values. It avoids the user to have think about separators and I believe given the flexibility of your script in matching patterns, three should be sufficient to capture all use-cases 
2. (Current) Use one entry field in the admin GUI. To enable multiple values in one string, a separator needs to be considered (now semicolon), and the script needs to build an array from splitting a string. Advantage is the user can configure as many as she likes, while the additional complexity is the separator 
3. Try to have the user input a string in javascript array notation, and then convert from string to array. While this also is a conversion to take place, in addition the array notation in a string is more complex and we should not expect users to be aware of javascript array notation format. 

The more I think about it actually I believe even 1) might be the best for the user and we can handle array creation in the script easily. I do recommend against 3)